experimental/traffic-portal/server.ts

```diff
@@ -29,15 +29,24 @@ import { AppServerModule } from "./src/main.server";
 
 let config: ServerConfig;
 
+const prodDistFolder = "/opt/traffic-portal/browser";
```

Comment:
I would request that this be the default value for a configurable dist folder like the configuration file location. Well, that or the CWD be the default and then the default prod configuration file specifies the dist folder to which the RPM installs.